types/utils_request.d.ts

```diff
@@ -0,0 +1,54 @@
+export const FIREFOX_MOBILE_USER_AGENT: "Mozilla/5.0 (Android; Mobile; rv:14.0) Gecko/14.0 Firefox/14.0";
+export const FIREFOX_DESKTOP_USER_AGENT: "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:68.0) Gecko/20100101 Firefox/68.0";
+export namespace REQUEST_AS_BROWSER_DEFAULT_OPTIONS {
+    export const countryCode: string;
+    export const languageCode: string;
+    export const headers: {};
+    export const method: string;
+    export const useMobileVersion: boolean;
+    export const useBrotli: boolean;
+    export const json: boolean;
+    export function abortFunction(res: any): boolean;
+    export const useCaseSensitiveHeaders: boolean;
+    export const useStream: boolean;
+    export const proxyUrl: any;
+    export const timeoutSecs: number;
+}
+export function requestAsBrowser(options: RequestAsBrowserOptions): Promise<any>;
+/**
+ * (@apify/http-request)[https://www.npmjs.com/package/@apify/http-request] NPM package.
+ */
+export type RequestAsBrowserOptions = {
+    /**
+     * URL of the target endpoint. Supports both HTTP and HTTPS schemes.
+     */
+    url: string;
+    /**
+     * HTTP method.
+     */
+    method?: string;
+    /**
+     * Additional HTTP headers to add. It's only recommended to use this option,
+     * with headers that are typically added by websites, such as cookies. Overriding
+     * default browser headers will remove the masking this function provides.
+     */
+    headers?: any;
+    /**
+     * Two-letter ISO 639 language code.
+     */
+    languageCode?: string;
+    /**
+     * Two-letter ISO 3166 country code.
+     */
+    countryCode?: string;
+    /**
+     * If `true`, the function uses User-Agent of a mobile browser.
+     */
+    isMobile?: boolean;
+    /**
+     * Function accepts `response` object as a single parameter and should return true or false.
+     * If function returns true request gets aborted. This function is passed to the
+     * (
```

Comment:
the description seems cut off

Reply:
From `utils_request.js`:
```
 * @property {Function} [abortFunction]
 *  Function accepts `response` object as a single parameter and should return true or false.
 *  If function returns true request gets aborted. This function is passed to the
 *  (@apify/http-request)[https://www.npmjs.com/package/@apify/http-request] NPM package.
```
As soon as typescript parser sees `@` (as in `@apify/http-parser`) anywhere in the comment, it starts parsing a JSDoc tag and throws away everything after that point. Can't help it right now, I'd better fix other things in the parser first.

Reply:
Wouldn't work even with a `<a href="...>...</a>`.

Reply:
so it's another bug from the JSDoc parser from TSC, argh